learn/wavemaker-release-notes/v10-13-2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "WaveMaker 10.13.2 - Release date: 11 April 2022"
+id: ""
+sidebar_label: "Release v10.13.2"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## Improvements
+---
+
+- FileUpload widget: **Clear** and **Delete** methods are introduced to clear and delete uploaded files. 
+- Introduced a new property to show characters limits for the Text widget. 
+
+## Bug fixes
+---
+
+**Spring Update**
+
+- Upgraded Spring dependency version to mitigate against the vulnerability [CVE-2022-22965](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-22965).
+
+- Spring upgraded to **5.3.18**.
+
+:::note
+To learn more about Spring dependency workaround, see [Mitigating Spring Vulnerability](/learn/blog/2022/04/05/mitigating-spring-zero-day-vulnerability).
+:::
+
+### Other Bugs
+
+- Fixed an issue related to REST API OAuth 2.0 pop-up, requesting to re-authenticate on a new tab. 
+- Fixed an issue related to the Autocomplete widget inside the Form, including the dropdown not showing in runtime when the web service with a Request body is bound to autocomplete widget.
+- Fixed an issue related to the Tree widget's function when placed inside the container. The `selectById ()` function is working as expected. 
+
```

Comment:
Many bug fixes are missing, pelase refer this list https://wavemaker.atlassian.net/issues/?filter=13578